jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -578,10 +574,9 @@ code_blockt get_thread_safe_clinit_wrapper_body(
 
   // if(init_complete) return;
   {
-    code_ifthenelset conditional;
-    conditional.cond() = init_complete.symbol_expr();
-    conditional.then_case() = code_returnt();
-    function_body.add(conditional);
+    const code_ifthenelset conditional(
+      init_complete.symbol_expr(), code_returnt());
+    function_body.add(std::move(conditional));
```

Comment:
No need for the temporary `conditional` any longer?

Reply:
True, but I still left it in place as the line otherwise got long-ish. But if anyone thinks it should rather be removed then I'll happily do so.

Reply:
BTW, the combination of 'const' and std::move won't do anything.